seg000.c

```diff
@@ -127,7 +127,6 @@ void __pascal far init_game_main() {
 	hof_read();
 	//sub_16C48();
 	//sub_16C1B();
-	method_7_alloc(-1);
```

Comment:
Just to clarify, i removed this because it was attempting to malloc '-1 bytes' (i've been compiling on VisualStudio 2013.. not sure if this was the cause or not, but either way... the compare of -1 >= 0xFFFE in this case, was failing...)

not really sure what the point of this call was anyway, as the malloc result isnt kept